vizzy/templates/ingress.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     "nginx.ingress.kubernetes.io/force-ssl-redirect": "true"
     "kubernetes.io/ingress.class": nginx
+    "nginx.ingress.kubernetes.io/proxy-body-size": "100m"
```

Comment:
Make this a configuration using `{{ .Values.ingress.proxyBodySize }}` and put it in the values.yaml as a default.

Reply:
and move it up one line